.github/workflows/ci.yml

```diff
@@ -125,6 +125,12 @@ jobs:
       - name: Compress workspace artifact
         run: tar --create --use-compress-program='zstd -T0' --file=/tmp/workspace.tar.zst .
 
+      - name: Generate Fathom files
+        run: yarn codegen:fathom
+        env:
+          FATHOM_TOKEN: ${{ secrets.FATHOM_TOKEN }}
+          FATHOM_SITE_ID: ${{ secrets.FATHOM_SITE_ID }}
+
```

Comment:
This step should be before 

- name: Compress workspace artifact
        run: tar --create --use-compress-program='zstd -T0' --file=/tmp/workspace.tar.zst .